scripts/build/unzip.ps1

```diff
@@ -1,15 +1,17 @@
 <#
     .SYNOPSIS
     This script wraps c#'s io.compression.zipfile to unzip a zip file.
-    The script is intended to be used within the VSO build to unzip the winobjc SDK.
+    The script is intended to be used within the VSO build to unzip a NuGet package.
 #>
 param(
-    #NOTE: using Parameter(Mandatory) will hang the VSO build job since powershell will block waiting for user input.
+    #NOTE: using Parameter(Mandatory) will hang the VSO build job since
+    #      powershell will block waiting for user input.
     [string]$zipFile = $(throw 'Mandatory parameter "zipFile" is not set.'),
```

Comment:
I'm not sure what the build def looks like, but, do we need to do this at all?  I thought we don't have  a zip file anymore.